model/CHANGELOG.md

```diff
@@ -2,6 +2,92 @@
 
 Changelog for `twilight-model`.
 
+## [0.5.0] - 2021-06-13
+
+### Upgrade Path
+
+Remove references to `Guild::nsfw`, `Guild::region`, `PartialGuild::nsfw`,
+`PartialGuild::region`, and `TemplateGuild::region`.
+
+Replace the following usages:
+```diff
+-twilight_model::channel::invite::TargetUserType
++twilight_model::channel::invite::TargetType
+
+-twilight_model::gateway::payload::update_status::UpdateStatus
++twilight_model::gateway::payload::update_presence::UpdatePresence
+
+-twilight_model::gateway::payload::update_status::UpdateStatusInfo
++twilight_model::gateway::payload::update_presence::UpdatePresencePayload
+```
+
+### Additions
+
+Support for Slash Commands has been added.
+
+The following models have been added:
+
+- `enum CommandDataOption`
```

Comment:
I think since the whole module is new these don't have to be individually listed as I'm not sure listing them adds value, what do you think?